### DISCUSSION TITLE
Est-ce qu'il est possible de mettre des sections avec du formatage niveau H2

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#167] [Annonces]
Bonjour,

J'ai un formulaire avec plusieurs sections comme codé en exemple ci-dessous:
section:
            fr: Information sur l'organisme        
            id: sectionRenseignementOrg
Est-ce qu'il est possible de mettre l'affichage à tire de niveau H2 ?

Merci

**Comment by loima02** [comment]
Bonjour,

Normalement, le h2 est utilisé pour le titre de section et le contenu du formulaire devrait débuter avec les h3.

Ceci dit, il est possible, bien que pas cute, d'utiliser une section dynamic et inscrire un h2 dedans.

Exemple :

````yaml
- name: pikachu
  type: dynamic
  text:
    fr: <h2>Titre en h2</h2>
````

![image](https://github.com/MTESSDev/FRW/assets/35601944/2c8902fc-f7b9-48ae-b802-9ff051b984b3)
